api/blockAPI_test.go

```diff
@@ -143,14 +137,18 @@ func getBlockHashes(jsonBlockObject []map[string]json.RawMessage) []types.Hash {
 }
 
 func TestBlockNumber(t *testing.T) {
-	db, err := connectToDatabase()
+	cfg, err := config.LoadConfig("../testing-resources/api_test_config.yml")
+	if err != nil {
+		t.Fatal("Error parsing config TestBlockNumber", "err", err.Error())
+	}
+	db, err := connectToDatabase(cfg)
 	if err != nil {
 		t.Fatal(err.Error())
 	}
 	defer db.Close()
-	cfg, err := config.LoadConfig("../testing-resources/api_test_config.yml")
-	if err != nil {
-		t.Fatal("Error parsing config", "err", err.Error())
+	for _, path := range cfg.Databases {
```

Comment:
It would be good to put this _before_ the defer db.Close(), because defers run first-in-last-out, so you'd be deleting the wal files before the database gets closed under this configuration.

Reply:
Take this into consideration in similar loops elsewhere.